api/v1beta1/helmchart_types.go

```diff
@@ -45,6 +53,15 @@ type HelmChartSpec struct {
 	// +required
 	Interval metav1.Duration `json:"interval"`
 
+	// Determines what enables reconciliation. Valid values are ('ChartVersion',
+	// 'Revision'). ChartVersion reconciles when the version of the Helm Chart
+	// is different. Revision reconciles when the Revision of the source is different.
```

Comment:
```suggestion
	// Determines what enables reconciliation. Valid values are ('ChartVersion',
	// 'Revision'). See the documentation of the values for an explanation on their
	// behavior.
```

Allows for growth, and is less maintenance intensive.